### ISSUE TITLE
Bug: /api/v1/nautical/start_backup doesn't respond until the backup is finished

### ISSUE THREAD

**Issue by Riewest** [Minituff/nautical-backup#209]
When calling the start_backup endpoint `/api/v1/nautical/start_backup` nautical doesn't respond until the containers have finished backing up. But the response just says it started.
```http
HTTP/1.1 200 OK
date: Mon, 22 Apr 2024 02:22:15 GMT
server: uvicorn
content-length: 50
content-type: application/json
Connection: close

{
  "message": "Nautical Backup started successfully"
}
```

Example Request I made took 23 seconds (All containers had already stopped and restarted)
![image](https://github.com/Minituff/nautical-backup/assets/16376061/05b74956-df2e-4cca-a4ad-3b7e84184270)

**Comment by Minituff** [comment]
Understood. This one should be an easy fix. Thanks for pointing it out. 

I may actually keep the current functionality but move it to a different endpoint because it may be or use to have the API only respond when the backup is complete.

**Comment by Riewest** [comment]
That makes sense to me.

**Comment by Riewest** [comment]
Can you have the api response include the list of containers it backed up easily? If not no worries

**Comment by Minituff** [comment]
Yeah I think I can do that too.

**Comment by Riewest** [comment]
Successful containers on complete is what I was thinking. Just another nice to have thing.

Thanks for all the work you do on this (And no rush on any of this, I'm just having fun using it and figuring out what I can do with it)

**Comment by Minituff** [comment]
Hey, this should be fixed in [Version 2.2.0](https://github.com/Minituff/nautical-backup/releases/tag/v2.2.0).

I added a new endpoint called `kickoff_backup` that should now respond immediately but run Nautical in the background.

I will also consider add another endpoint in the future that has more info about the containers backed up/errors/skips/etc.
